context-free.md

```diff
@@ -2,21 +2,27 @@
 
 ##  Who is this programming language for?
 
+Context Free is for anyone artistically inclined who either has programming experience or is willing to learn some basic programming concepts.
 
 ## What is easy to do in this language? Why is it easy?
 
+In Context Free it's very easy to draw images involving patterns of simple geometric shapes.  By combining and transforming squares, circles, and triangles, most shapes can be created in a relatively straightforward manner.  Once you've created your shapes, Context Free's looping syntax allows you to easily create patterns with those shapes.
```

Comment:
I think its also important to recognize one of the important features of the language: recursion